GetUserDataProcessingActivity/handler.ts

```diff
@@ -0,0 +1,204 @@
+/**
```

Comment:
Do we need this Activity ? I'm asking because the orchestrator will be triggered by an upcoming record inserted into user-data-requests collection so the input comes from the orchestrator client function bindings

Reply:
(it's ok to use a query in the client.ts)

Reply:
I basically did it for not have to construct here models and require env keys (as we do already with other activities). I guess we can eventually remove it if we find it unhelpful in future refactoring

Reply:
yes we should convert this into an utility function only used by the cli (not by the orchestrator)